tests/installation/partitioning_warnings.pm

```diff
@@ -38,7 +38,7 @@ sub process_missing_special_partitions {
     if (is_storage_ng && check_var('ARCH', 's390x')) {    # s390x needs /boot/zipl on ext partition
         process_warning(warning => 'no-boot-zipl', key => 'alt-n');
     }
-    elsif (get_var('OFW')) {                              # ppc64le needs PReP /boot
+    elsif (check_var('ARCH', 'ppc64le')) {                # ppc64le needs PReP /boot
```

Comment:
I am sure you are aware of the history of this code but unfortunately your git commit message is not explaining to me any good reasoning. So I have to ask. Are you *sure* you want to revert 8e30d7c02 here? Doesn't this break ppc64 (BE) again?

Reply:
I doubt that this test suite is executed anywhere else. So for this particular case we are safe. To my knowledge those are synonyms.